### PR TITLE
Create PopUp 

### PR DESCRIPTION
**Pull request by Dalmano**
Relates #9 used react-bootstrap modal for accessibility. 

const [modalShow, setModalShow] = React.useState(false);

   <button onClick={() => setModalShow(true)}>Click me</button>

      <PopUp
        show={modalShow}
        content="Thank you, your response has been successful"
        onHide={() => setModalShow(false)}
      />

Use the code above to test it. Waiting on the merging of the button to finish this P.R.

### CODE REVIEW

**Comment by dubhcait** [comment]
actual can use. https://material-ui.com/components/modal/

**Comment by Dalmano** [comment]
incomplete, need to add the 'Ok' button

**Comment by AbdallahAmmar96** [comment]
pull the master and solve the conflict @Dalmano

File: client/src/components/PopUp/index.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import React from "react";
```
Comment (AbdallahAmmar96) [inline client/src/components/PopUp/index.js]: Add a new line after import third-party modules and then put import the style file  
```suggestion
import React from "react";
import Dialog from "@material-ui/core/Dialog";
import DialogActions from "@material-ui/core/DialogActions";
import Button from "./../Button";
import DialogTitle from "@material-ui/core/DialogTitle";

import "./style.css";
```

**Pull request closed without merging**